xml/depl_crowbar.xml

```diff
@@ -0,0 +1,368 @@
+    <para> On any machine, start a Web browser and make sure that JavaScript and
+     cookies are enabled.</para>
+   </step>
+   <step>
+    <para> As URL, enter the IP address of the &admserv;, for example:</para>
+    <screen>http://192.168.124.10/</screen>
+   </step>
+   <step>
+    <para>Log in as user
+     <systemitem class="username">crowbar</systemitem>. If you have not changed
+     the password, it is <literal>crowbar</literal> by default.
+    </para>
+   </step>
+  </procedure>
+  <!--taroth 2016-02-05: https://bugzilla.suse.com/show_bug.cgi?id=950482-->
+  <procedure xml:id="pro.depl.crow.password">
+   <title>Changing the Password for the &crow; Web Interface</title>
+   <step>
+    <para>After being logged in to the &crow; &wi;, select <menuchoice>
+      <guimenu>&Barcl;</guimenu>
+      <guimenu>Crowbar</guimenu>
+     </menuchoice>.</para>
+   </step>
+   <step><para>Select the <literal>&crow;</literal> &barcl; entry and
+    <guimenu>Edit</guimenu> the proposal.</para>
+   </step>
+   <step>
+    <para>In the <guimenu>Attributes</guimenu> section, click
+      <guimenu>Raw</guimenu> to edit the configuration file.</para>
+   </step>
+   <step><para>Search for the following entry:</para>
+    <screen>"crowbar": {
+     "password": "crowbar"</screen>
+    </step>
+   <step>
+    <para>Change the password and <guimenu>Save</guimenu> the configuration.
+     <remark>taroth 2016-02-05: DEVs, is saving enough? Or do I need to redeploy
+      the changed Crowbar barclamp or to restart any services to apply the
+      change?</remark>
+    </para>
+   </step>
+  </procedure>
+ </sect1>
+
+ <sect1 xml:id="sec.depl.crow.overview">
+  <title>Overview: Main Elements</title>
+
+  <para>After logging in to &crow;, you will see a navigation bar at the
+   top-level row. Its menus and the respective views are described in the
+   following sections.</para>
+  <figure>
+   <title>&crow; UI&mdash;&dash; (Main Screen)</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="crowbar_main_dashboard.png" width="100%"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="crowbar_main_dashboard.png" width="100%"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+  
+  <sect2 xml:id="sec.depl.crow.overview.nodes">
+   <title>Nodes</title>
+   <variablelist>
+    <varlistentry>
+     <term><guimenu>&dash;</guimenu></term>
+     <listitem>
+      <para>This is the default view after logging in to the &crow;
+       &wi;. The &dash; shows the groups (which you can create to
+       arrange nodes according to their purpose), which nodes belong to each
+       group and which state the nodes and groups are in. In addition, the
+       total number of nodes is displayed in the top-level row.</para>
+      <para>The color of the dot in front of each node indicates the node
+       status:</para>
+      <itemizedlist>
+       <listitem>
+        <para>Gray means the node is being discovered by the
```

Comment:
Please note that gray is also used when there's no up-to-date information about a deployed node (which typically means that chef-client is not running anymore on the node).